dll/win32/shell32/shlfolder.cpp

```diff
@@ -54,6 +54,89 @@ BOOL SHELL_IncludeItemInFolderEnum(IShellFolder *pSF, PCUITEMID_CHILD pidl, SFGA
     return TRUE;
 }
 
+static HRESULT
+MapSCIDToShell32FsColumn(const SHCOLUMNID *pscid, VARTYPE &vt)
+{
+    if (pscid->fmtid == FMTID_Storage)
+    {
+        switch (pscid->pid)
+        {
+            case PID_STG_NAME:        return (vt = VT_BSTR, SHFSF_COL_NAME);
```

Comment:
Should be the same without weird syntax.
```suggestion
            case PID_STG_NAME:        vt = VT_BSTR; return SHFSF_COL_NAME;
```